gpflow/kernels/conditioned.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+import tensorflow as tf
+from .base import Kernel
+from ..config import default_float, default_jitter
+
+
+class Conditioned(Kernel):
+    """
+    Conditioned kernel. Can be used to wrap any kernel
+    to transform it into a its conditioned version.
+
+    This provides a simple way to condition a Gaussian process on noiseless observations
+
+    From a covariance function k(.,.) and a conditioning dataset (Xc, Yc),
+    a new covariance function and a conditional mean function
+    are created as
+
+       kc(.,.) = k(.,.) - k(.,Xc)k(Xc, Xc)⁻¹k(Xc,.)
+       mc(.)   = k(.,Xc)k(Xc, Xc)⁻¹Yc
+    """
+
+    def __init__(self, base: Kernel, xc: tf.Tensor, yc: tf.Tensor):
```

Comment:
The `xc` and `yc` are very obscure names. Could you call them differently to reflect their meaning?

Reply:
renamed!